lib/Sema/ConstraintSystem.cpp

```diff
@@ -879,15 +879,7 @@ Optional<Type> ConstraintSystem::isSetType(Type type) {
 }
 
 bool ConstraintSystem::isCollectionType(Type type) {
```

Comment:
Do we still need `ConstraintSystem::isCollectionType` if it's already a method on `TypeBase`?